Arrays/2779MaximumBeautyofanArrayAfterApplyingOperation.py

```
@@ -0,0 +1,62 @@
+                result+=list(range(0,right+1,1))
+            else:
+                result+=list(range(left,right+1,1))
+        hash_map=defaultdict(int)
+        for element in result:
+            hash_map[element]+=1
+        # print(hash_map)
+        result=list(hash_map.items())
+        # print(result)
+        result.sort(reverse=True,key=lambda x:x[1])
+        return result[0][1]     
+class Solution:
```

Flake8 (7.2.0)

[error] redefinition of unused 'Solution' from line 20

(F811)